webapp/apps/taxbrain/helpers.py

```diff
@@ -162,8 +165,8 @@ def string_to_float_array(s):
     'df_dec': 'Difference between Base and User plans by expanded income decile',
     'mX_bin': 'Base plan tax vars, weighted avg per expanded income bin',
     'mY_bin': 'User plan tax vars, weighted avg per expanded income bin',
-    'df_bin': 'Difference between Base and User plans by expanded income bin',
-    'fiscal_tots': 'Total Revenue Change by Calendar Year',
+    'df_bin': 'Individual Income Tax: Difference between Base and User plans by AGI decile',
```

Comment:
Hmm.. something has gone wrong here. Any name that ends with `_bin` refers to data that has been grouped by an income bin. Here, you are saying that `df_bin` is referring to data that is grouped by AGI_decile. If I understand correctly, the line should be:
```
'df_bin': 'Individual Income Tax: Difference between Base and User plans by expanded income bin',
```

@Amy-Xu do you see what I mean?